### DISCUSSION TITLE
Why does "flex column justify-between" not work on my QCardSection?

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#14803] [General - Components / Directives / etc] [answered]
I have a page with three QCards. Each card has some QCardSection's within.

And one of those cards has some QLinearProgress bars. It looks like this:

<img width="352" alt="Screenshot 2022-11-04 at 9 51 18 am" src="https://user-images.githubusercontent.com/2043257/199875985-15f69aaf-8823-4d2b-9220-37992ad89c4a.png">

And now I am trying to space out those progress bars vertically using `flex column justify-between` on the QCardSection.

But that is having no effect. Nothing changes. They are not spaced out at all.

To solve this problem I tried also adding `fit` like so `fit flex column justify-between`. And that spaced them out, but now the progress bars go way beyond the QCard itself, and they are hidden behind another QCard below. See here:

<img width="356" alt="Screenshot 2022-11-04 at 9 51 08 am" src="https://user-images.githubusercontent.com/2043257/199876322-6747ba80-e785-4086-b9e2-30356e502982.png">

This behavior is very confusing. Any help would be very much appreciated.

How do I use `flex` to space them out vertically within the card?

Here is a working reproduction showing the most important parts: https://stackblitz.com/edit/quasarframework-mtabwv?file=src/components/AuditSummaryCard.vue

**Comment by suleimanas** [comment, marked as answer]
https://stackblitz.com/edit/quasarframework-qxmulm?file=src/components/AuditSummaryCard.vue

2 changes:
1. `<q-card flat bordered class="fit column">`
2. `<q-card-section class="col column justify-between">`

**Reply by BenJackGill** [reply]
Thanks! This works, but I'm not sure why...

I understand the need to put `column` on the QCard. Because that would stack everything vertically.

But why does the QCardSection need `col` to make the inner parts of that card obey they flex rules? I don't get that part.

**Reply by suleimanas** [reply]
about .col https://quasar.dev/layout/grid/column#all-breakpoints
happy to help :)
